src/components/tl/mlx5/mcast/tl_mlx5_mcast_helper.c

```diff
@@ -401,14 +401,14 @@ ucc_status_t ucc_tl_mlx5_fini_mcast_group(ucc_tl_mlx5_mcast_coll_context_t *ctx,
 
     dst = inet_ntop(AF_INET6, &comm->mcast_addr, buf, 40);
     if (NULL == dst) {
-        tl_error(comm->lib, "inet_ntop failed");
+        tl_warn(comm->lib, "inet_ntop failed");
```

Comment:
here also, there is no fallback, right? If yes it should be reported as an actual error.
Please revert this change where an error should be reported (i.e. when the system doesn't internally fall back on another mechanism)